nearcore/res/runtime_configs/29.json

```diff
@@ -0,0 +1,180 @@
+{
```

Comment:
Should we store the diff for each version instead of the whole file?

Currently, only `with_lower_storage_cost` changes between different version. Storing the whole file makes it hard to see what changed between different versions and can be error prone because of that. 

We should also add regression tests to make sure existing runtime_configs do not get changed once they are added.

Reply:
That's a good question.

I would vote for storing the whole files, because it improves readability - in the future, when we may have > 10 configs, it will be much easier to find the exact value for some protocol version, comparing with looking at all previous versions.

Also, I feel that applying diffs one-by-one is error prone in different sense - we never see the config which is actually used in production. So we would need the whole configs at least for testing.

For comparing configs, we already have a tool:

```
python3 ./runtime/runtime-params-estimator/emu-cost/compare_costs.py ./nearcore/res/runtime_configs/29.json ./nearcore/res/runtime_configs/42.json --diff
```

Before adding new config, I suggest running this tool and attaching the result to corresponding PR. Though at first I need to improve its usability a bit. 

What do others think? @matklad @bowenwang1996 

> We should also add regression tests to make sure existing runtime_configs do not get changed accidentally once they are added.

Yes, that's right.

Reply:
I discussed the idea of storing diffs with @matklad before. We agreed that it felt like overengineering at this point. Also it is very likely we change many config values in the same PR (we will likely do this once we are ready to update the costs), which would make storing diffs less useful.